python/lsst/daf/butler/core/_butlerUri/http.py

```diff
@@ -84,11 +84,22 @@ def getHttpSession() -> requests.Session:
 
     log.debug("Creating new HTTP session...")
 
+    ca_bundle = None
+    try:
+        ca_bundle = os.environ['LSST_BUTLER_WEBDAV_CA_BUNDLE']
+    except KeyError:
+        log.warning("Environment variable LSST_BUTLER_WEBDAV_CA_BUNDLE is not set: "
```

Comment:
You are right, but it is not specifically about webdav neither. I changed the statement to make it clearer that it is mostly useful to trust non-default CAs, and made it a debug log instead of a warning log.